### PR TITLE
Autodoc bonesetting

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
```SUMMARY: Features "Added ability to splint broken limbs to autodoc. Replaced computer bonesetting with autodoc bonesetting."```

#### Purpose of change
Get rid of weird behavior when computer somehow splints broken limbs.

#### Describe the solution
- Copied the logic of bonesetting from `computers.cpp` into `iexamine::autodoc`.
- Splinting don't require anesthesia, so I had to modify `needs_anesthesia` bool to check for >1 choice in autodoc menu.
- Added ability to splint not only player's limbs, but also NPC's limbs.
- Added a message if no limbs require splinting.
- Replaced bonesetting computers in hospitals' and labs' mapgen with autodoc and its couch.
- Removed instances of `COMPACT_BONESETTING` computer action.

#### Describe alternatives you've considered
None.

#### Additional context
At the moment autodoc can splint unlimited number of broken limbs and seems to take splints out of thin air. This is the original (computer one) behavior, but I thought I could change that by manually placing (in jsons) some arbitrary number - say, 4 - of splints in autodoc and making autodoc sealed container. Each splinting will consume one splint, so after 4 splintings it will stop providing bonesetting treatment.
But I don't know if we need this limitation at all, and if we do, if my solution if acceptable.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Comment by curstwist** [comment]
Did you catch the lab bonesetter?  Will the nursebot effect this any?  Good idea : )

**Comment by Night-Pryanik** [comment]
> Did you catch the lab bonesetter?

Yeah, I did.

> Will the nursebot effect this any?
No, it won't

**Comment by tinukedaya** [comment]
> At the moment autodoc can splint unlimited number of broken limbs and seems to take splints out of thin air. This is the original (computer one) behavior, but I thought I could change that by manually placing (in jsons) some arbitrary number - say, 4 - of splints in autodoc and making autodoc sealed container. Each splinting will consume one splint, so after 4 splintings it will stop providing bonesetting treatment.
> But I don't know if we need this limitation at all, and if we do, if my solution if acceptable.

What if you make the autodoc splint be some kind of 3d-printed brace? This kind of thing >>> 
image

Autodoc can have a supply of the material (or maybe require plastic chunks for bonesetting). Also the Autodoc brace might be less encumbering (and maybe even not reset the healing status on some minor damage to the limb) thus the player might actually seek the autodoc for bonesetting? 

Should be a one time use item. When you take it off, it will be converted to plastic chunks.

**Comment by kevingranade** [comment]
I think we eventually want some kind of substance to construct the splint from, but no need at the moment.

The autodoc splint is already better than the player-applied one.

**Comment by Night-Pryanik** [comment]
While testing, I found out that we definitely need a way to say "Go to this tile" to our NPC followers so they could lay up on the autodoc couch - they consider it a hard to cross terrain and try to avoid it at all costs.

File: src/computer.h [hunk 1]
```diff
@@ -41,7 +41,6 @@ enum computer_action {
     COMPACT_BLOOD_ANAL,
     COMPACT_DATA_ANAL,
     COMPACT_DISCONNECT,
-    COMPACT_BONESETTING,
```
Comment (nexusmrsep) [inline src/computer.h]: Will this not break the logic of existing  computers in savegames?

Comment (Night-Pryanik) [reply src/computer.h]: Tested right now. The game don't crash, no errors popping up. Existing medical computers won't be working, of course, activating them will show "evacuation" message. I think that's acceptable.

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Pull request merged**